chroma_core/lib/graphql.py

```diff
@@ -0,0 +1,32 @@
+import os
+import requests
+import settings
+from django.db import connection
+
+
+API_USER = None
+API_KEY = None
+
+
+def get_api_session_request():
+    global API_USER
+    global API_KEY
+    if not API_USER or not API_KEY:
```

Comment:
nit: tuple of `API_CREDS = (user, key)` would be a bit easier to work with

Reply:
> I think this looks good. Has it been tested yet for both rpm and docker?

I've manually tested the logic on RPM system.